conf/broker.conf

```diff
@@ -960,6 +960,10 @@ managedLedgerMinLedgerRolloverTimeMinutes=10
 # Maximum time before forcing a ledger rollover for a topic
 managedLedgerMaxLedgerRolloverTimeMinutes=240
 
+# Time to rollover ledger for inactive topic (duration without any publish on that topic)
+# Disable rollover with value 0 (Default value 0)
+managedLedgerInActiveLedgerRolloverTimeMinutes=0
```

Comment:
Nit: inactive is used as a single word here. I think the casing should be `Inactive` here and in the rest of the code base.

Reply:
done